core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorTableMetadata.java

```diff
@@ -72,13 +83,24 @@ public Optional<String> getComment()
         return comment;
     }
 
+    /**
+     * Engine would not violate the check constraint when predicate evaluates to UNKNOWN (e.g. NULL > 100).
```

Comment:
Start with saying what it is

```suggestion
     * List of constraints data in a table is expected to satisfy. 
     * Engine ensures rows written to a table meet these constraints.
     * Engine would not violate the check constraint when predicate evaluates to UNKNOWN (e.g. NULL > 100).
```

(plus reformat)

Reply:
Or we can say something like what the SQL spec says:

"A check constraint is satisfied when it evaluates to True or Unknown"